.github/workflows/regression.yml

```diff
@@ -61,9 +61,9 @@ jobs:
         uses: actions/upload-artifact@v3
         with:
           path: output/${{ steps.vars.outputs.tgz }}
+
       - name: Regression Test
-        run: |
-          rc=true
-          make test-qeneth || rc=false
-          cat test/.log/last/result.md >> $GITHUB_STEP_SUMMARY
-          eval $rc
+        run: make test-qeneth
+
+      - name: Publish Test Result
+        run: cat test/.log/last/result-gh.md >> $GITHUB_STEP_SUMMARY
```

Comment:
Neat with more fancy log file! :rocket: 

However, all GitHub actions run with a sort of `set -e`, so with your refactor we'll never get the test results published if any test fails.

Reply:
> Neat with more fancy log file! 🚀
> 
> However, all GitHub actions run with a sort of `set -e`, so with your refactor we'll never get the test results published if any test fails.

Great point. Didn't know that. I added the magical GitHub syntax `if: always()` which ensures the test results are published even if previous jobs fails.

Here's an example from the test framework:
https://github.com/rical/9pm/actions/runs/6377739586

Reply:
Nice!  Now I can fix a few of my other actions as well, thanks :smiley::+1: